src/NavBar/EventsButton.jsx

```diff
@@ -64,24 +42,37 @@ export class EventsButton extends Component {
 	}
 
 	render() {
-		const iconClasses = `${this.props.classes.newEventIcon} ${
-			this.state.isHovering ? this.props.classes.newEventIconHover : null
+		const { classes, label } = this.props;
+		const iconClasses = `${classes.newEventIcon} ${
+			this.state.isHovering ? classes.newEventIconHover : null
 		}`;
-		const icon =
-			this.state.nextEvent !== '' ? (
-				<NotificationImportantIcon className={iconClasses} />
-			) : null;
+		let icon = null;
+		let nextEventText = '';
+		if (this.props.nextEvent) {
+			icon = <span className={iconClasses}> ●</span>;
+			const locale = i18n.language === 'nl' ? 'nl-NL' : 'en-US';
+			const formattedDate = new Date(
+				this.props.nextEvent.time
+			).toLocaleDateString(locale, { month: 'long', day: 'numeric' });
+			nextEventText = `${formattedDate} *Meetup* ${this.props.nextEvent.name}`;
```

Comment:
Personally I'm not a big fan of using `let`s. At Pronto we prefer using inline if-else statements or just functions that return the value as `const`. But I guess this is just a personal preference!